scenario_player/tasks/services.py

```diff
@@ -358,3 +357,4 @@ def _process_response(self, response_dict: dict):
                 raise ScenarioAssertionError(
                     f"Expected amount of {exp_iou_amount} but got {actual_iou_amount}."
                 )
+        return response_dict
```

Comment:
We should add type annotations for this. 

@ulope Do/should those always return dicts, or can it be any value?

Reply:
IIRC `_process_response` should always return a dict because another subclass might want to further process it.

Reply:
I'll fix that in a follow-up.